src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -111,7 +111,7 @@ def _flexible_server_create(cmd, client, resource_group_name=None, server_name=N
                        ' reset your password with \'az mysql flexible-server update -n %s -g %s -p <new-password>\'.',
```

Comment:
Dont see the _flexible_list_skus() function in the mysql file that you are referencing in the sku-list command for mysql. Missed it ?

Reply:
I haven't made that part sine that this command was not deployed yet. I'll add it to the next commit!